src/WebForms/SiteMapOptions.cs

```diff
@@ -0,0 +1,12 @@
+// MIT License.
+
+using System.Configuration;
+
+namespace System.Web;
+
+public class SiteMapOptions
+{
+    public string DefaultProvider { get; set; }
+    public bool? Enabled { get; set; }
+    public ProviderSettingsCollection Providers { get; set; }
```

Comment:
change this to an ICollection<> - you can have ProviderSettingsCollection behind the scenes implement it, but that class is in System.Configuration